src/hooks/useSubStep.ts

```diff
@@ -0,0 +1,51 @@
+import {ReactNode, useCallback, useRef, useState} from 'react';
+
+type Props = {
+    bodyContent: ReactNode[];
+    onFinished: () => void;
+    startFrom?: number;
+};
+
+export default function useSubStep({bodyContent, onFinished, startFrom = 0}: Props) {
```

Comment:
I think you have to type the returning https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#type-inference

Reply:
I had the return typed but then noticed the inference was spot on so removed it as imo it's just creating extra noise, quite a number of other hooks don't have the return typed tho.